libraries/abstractions/pkcs11/utest/CMakeLists.txt

```diff
@@ -0,0 +1,142 @@
+project ("abstractions pkcs11 cmock unit test")
+cmake_minimum_required (VERSION 3.13)
+set(3rdparty_dir "${AFR_ROOT_DIR}/libraries/3rdparty")
+set(abstractions_dir "${AFR_ROOT_DIR}/libraries/abstractions")
+set(standard_dir "${AFR_ROOT_DIR}/libraries/freertos_plus/standard")
+set(kernel_dir "${AFR_ROOT_DIR}/freertos_kernel")
+set(common_dir "${AFR_ROOT_DIR}/libraries/c_sdk/standard/common")
+
+# Mock library
+list(APPEND mock_list
+    "${abstractions_dir}/pkcs11/include/iot_pkcs11_pal.h"
+    "${kernel_dir}/include/portable.h"
+    "${kernel_dir}/include/queue.h"
+    "${common_dir}/include/iot_logging_task.h"
+    "${standard_dir}/crypto/include/iot_crypto.h"
+    "${3rdparty_dir}/mbedtls/include/mbedtls/platform_util.h"
+    "${3rdparty_dir}/mbedtls/include/mbedtls/ctr_drbg.h"
+    "${3rdparty_dir}/mbedtls/include/mbedtls/sha256.h"
+    "${3rdparty_dir}/mbedtls/include/mbedtls/base64.h"
+    "${3rdparty_dir}/mbedtls/include/mbedtls/bignum.h"
+    "${3rdparty_dir}/mbedtls/include/mbedtls/entropy.h"
+    "${3rdparty_dir}/mbedtls/include/mbedtls/md.h"
+    "${3rdparty_dir}/mbedtls/include/mbedtls/rsa.h"
+    "${3rdparty_dir}/mbedtls/include/mbedtls/ecp.h"
+    "${3rdparty_dir}/mbedtls/include/mbedtls/ecdsa.h"
+    "${3rdparty_dir}/mbedtls_utils/mbedtls_error.h"
+    "${3rdparty_dir}/mbedtls/include/mbedtls/pk.h"
+    "${standard_dir}/utils/include/iot_pki_utils.h"
+    )
+
+create_mock_list(pkcs11_mbedtls_mock 
```

Comment:
I will make this change to the secure sockets, if you could make them here too it would be great
set the names of the variables at the top module_name${MOCK_PREFIX} | ${REAL_PREFIX} | ${UTEST_PREFIX} something like thatr

I will make it even more simplified, but this could be a good start

Reply:
okay, i'll wait for you to push your change and then copy it